cli/text2embeddings.py

```diff
@@ -74,6 +74,10 @@ def encode_indexer_input(
 @click.argument(
     "output-dir",
 )
+@click.argument(
+    "files_to_parse",
+    required=False,
+)
```

Comment:
I think this needs to be a `click.option`, otherwise click doesn't know whether the third argument is this or an option?

Reply:
so, it works if all options are specified before the arguments, but you make a decent point

Reply:
I'll change it